xml/admin_ceph_gateway.xml

```diff
@@ -1546,6 +1546,50 @@ GET /<replaceable>BUCKET</replaceable>?mdsearch
     </variablelist>
    </sect3>
   </sect2>
+
+  <sect2 xml:id="archive-sync-module">
+   <title>Archive Synchronization Module</title>
+   <para>
+    The <emphasis>archive module</emphasis> utilizes the versioning feature of
+    S3 objects in &ogw;. You can configure an <emphasis>archive zone</emphasis>
+    that captures the different versions of S3 objects as they occur over time
+    in other zones. The history of versions that the archive zone keeps can
+    only be eliminated via gateways associated with the archive zone.
+   </para>
+   <para>
+    With such architecture, several non-versioned zones can mirror their data
+    and metadata via their zone gateways providing high availability to the end
+    users, while the archive zone captures all the data updates to consolidate
+    them as versions of S3 objects.
+   </para>
+   <para>
+    By including the archive zone in a multi-zone configuration, you gain the
+    flexibility of an S3 object history in one zone while saving the space that
+    the replicas of the versioned S3 objects would consume in the remaining
+    zones.
+   </para>
+   <sect3 xml:id="archive-sync-module-configuration">
+    <title>Configuration</title>
+    <tip>
+     <title>More Information</title>
+     <para>
+      Refer to <xref linkend="ceph-rgw-fed"/> for details on configuring
```

Comment:
link to the sync module section is also needed here http://docserv.nue.suse.com/documents/SES_6/ses-admin/single-html/#ceph-rgw-sync

Reply:
link added